src/libraries/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -329,7 +328,10 @@ public async Task SendReceive_VaryingLengthBuffers_Success(Uri server)
             using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
             {
                 var rand = new Random();
-                var ctsDefault = new CancellationTokenSource(TimeOutMilliseconds);
+
+                // It seems that sometimes the default timeout is not enough for browser so we will extend it
+                // See issue https://github.com/dotnet/runtime/issues/46909
+                var ctsDefault = PlatformDetection.IsBrowser ? new CancellationTokenSource(BrowserTimeOutMilliseconds) : new CancellationTokenSource(TimeOutMilliseconds);
```

Comment:
Is there harm in merging the 2 timeouts and use 30s also for non-Browser?

Reply:
Just did not want change existing functionality.

Reply:
Can we do this in a follow up PR?  Really waiting for this to be approved as have another PR waiting on this one.

Reply:
addressed in https://github.com/dotnet/runtime/pull/47906